code/modules/mob/living/carbon/alien/humanoid/alien_powers.dm

```diff
@@ -288,29 +287,51 @@ Doesn't work on other aliens/AI.*/
 	name = "Sneak"
 	desc = "Blend into the shadows to stalk your prey."
 	active = 0
-
 	action_icon_state = "alien_sneak"
 
 /obj/effect/proc_holder/alien/sneak/fire(mob/living/carbon/alien/humanoid/user)
 	if(!active)
-		user.alpha = 75 //Still easy to see in lit areas with bright tiles, almost invisible on resin.
-		user.sneaking = 1
-		active = 1
+		user.alpha = 25
+		user.sneaking = TRUE
+		active = TRUE
 		to_chat(user, "<span class='noticealien'>You blend into the shadows...</span>")
 	else
 		user.alpha = initial(user.alpha)
-		user.sneaking = 0
-		active = 0
+		user.sneaking = FALSE
+		active = FALSE
 		to_chat(user, "<span class='noticealien'>You reveal yourself!</span>")
 
+#define QUEEN_CALL_TIME 15 MINUTES
+
+/obj/effect/proc_holder/alien/call_shuttle
+	name = "Summon Shuttle"
+	desc = "Sends a fake message which will beckon an emergency shuttle to the station."
+	action_icon_state = "alien_sneak"
+	var/time_created
```

Comment:
```suggestion
	COOLDOWN_DECLARE(queen_call_time)
```

Reply:
I'm axing this feature in favor of something else, I'll make those adjustments along with these fixes so we don't have this problem.